packages/components/src/mobile/global-styles-context/utils.native.js

```diff
@@ -120,31 +120,62 @@ export function getBlockColors(
 	return blockStyles;
 }
 
-export function parseColorVariables( styles, colorPalette ) {
+export function parseVariables( styles, mappedValues ) {
```

Comment:
Instead of `parseVariables` should we rename it `parseStyleVariables` ?
To be a bit more specific here.

Reply:
Sounds good! I'll change it to that name. Thanks!